Ripple data API alternatives (#1113)
XRPLF/xrpl-dev-portal: content/references/data-api.md

**Warning:** The Ripple Data API v2 is deprecated with no ongoing support. Please use the native [XRP Ledger HTTP APIs](rippled-api.html) instead.

For information on the old Data API, see the [rippled-historical-database repository](https://github.com/ripple/rippled-historical-database).

## Alternatives

> Most common operations like requesting account information and balances, transaction history, etc. 
> can be sent to self hosted or public XRP Ledger nodes. Data can be retrieved using a [WebSocket connection](get-started-using-http-websocket-apis.html#websocket-api) or [JSON RPC (HTTP POST)](.org/get-started-using-http-websocket-apis.html#json-rpc).

Reviewer: ```suggestion
For most common operations, like requesting account balances or transaction history, you can query a self-hosted or [public XRP Ledger server](public-servers.html) using a [WebSocket connection](get-started-using-http-websocket-apis.html#websocket-api) or [JSON-RPC (HTTP POST)](get-started-using-http-websocket-apis.html#json-rpc).
```
